processes/protocol-upgrade.md

```diff
@@ -10,13 +10,13 @@ Ethereum has regular protocol upgrades that introduce performance, security and
 
 ### General
 - Upgrades must not be scheduled for major holidays or events.
-- Scheduling multiple upgrades in advance should not be done, and bundling the mainnet upgrade with other upgrades must not be done.
-    - If multiple upgrades are scheduled in advance, each testnet upgrade but must be at least 14 days apart.
-    - In the event that multiple testnet upgrades are scheduled, and the first a testnet upgrade is not deemed a success, the next testnet upgrade is automatically cancelled. The next testnet must not be scheduled until the first incident has been resolved, at which point the earliest point in time is 14 days from the ACD call it was agreed to proceed.
+- Bundling the mainnet upgrade with other upgrades should not be done.
+- If multiple testnet upgrades are scheduled in advance, each testnet upgrade but must be at least 10 days apart, ideally aiming for two weeks between upgrades.
```

Comment:
grammar nit.  is upgrade a noun or a verb here.  

```suggestion
- If multiple testnet upgrades are scheduled in advance, each testnet upgrade must be at least 10 days apart, ideally aiming for two weeks between upgrades.
```

if a verb, perhaps `may upgrade, but must`